### PR TITLE
{geo}[foss/2022a] MITgcmutils v0.1.2 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MITgcmutils/MITgcmutils-0.1.2-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+single dynamical kernel can be used to simulate flow of both the atmosphere 
+and ocean. The model is developed to perform efficiently on a wide variety of 
+computational platforms."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('tqdm', '4.64.0'),
```
Comment (lexming) [inline easybuild/easyconfigs/m/MITgcmutils/MITgcmutils-0.1.2-foss-2022a.eb:23]: I do not see any dependency on tqdm, is this really needed?

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/MITgcmutils/MITgcmutils-0.1.2-foss-2022a.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/m/MITgcmutils/MITgcmutils-0.1.2-foss-2022a.eb]: You are correct. Apologies for that, I have removed the surplus `tqdm`

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @sassy-crick !

**Pull request merged**